_posts/2020-06-29-release-1.11.0.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: post 
+title:  "Apache Flink 1.11.0 Release Announcement" 
+date: 2020-06-29T08:00:00.000Z
+categories: news
+authors:
+- morsapaes:
+  name: "Marta Paes"
+  twitter: "morsapaes"
+
+excerpt: The Apache Flink community is proud to announce the release of Flink 1.11.0! This release marks the first milestone in realizing a new vision for fault tolerance in Flink, and adds a handful of new features that simplify (and unify) Flink handling across the API stack. In particular for users of the Table API/SQL, this release introduces significant improvements to usability and opens up completely new use cases, including the much-anticipated support for Change Data Capture (CDC)! A great deal of effort has also gone into optimizing PyFlink and ensuring that its functionality is available to a broader set of Flink users.
+---
+
+The Apache Flink community is proud to announce the release of Flink 1.11.0! This release marks the first milestone in realizing a new vision for fault tolerance in Flink, and adds a handful of new features that simplify (and unify) Flink handling across the API stack. In particular for users of the Table API/SQL, this release introduces significant improvements to usability and opens up completely new use cases, including the much-anticipated support for Change Data Capture (CDC)! A great deal of effort has also gone into optimizing PyFlink and ensuring that its functionality is available to a broader set of Flink users.
```

Comment:
I have tried describing some release highlights with a bit more detail, and add a bit more excitement. Happy to hear your feedback:

```suggestion
The Apache Flink community is proud to announce the release of Flink 1.11.0! This release marks a major milestone for many areas of Flink. We are particularly excited about these highlights in this release:
- The **core engine** is introducing a major change to improve the checkpointing performance under heavy backpressure: **unaligned checkpoints** allow barriers to overtake in-flight records so that the checkpointing is not blocked by backpressure anymore.
- We are introducing a **new Source API** for the **DataSet API**, simplifying the implementation of (custom) sources by handling internals such as event-time handling, watermark generation or idleness detection by Flink. Sources build on the new API are usable for both batch and streaming applications.
- **Flink SQL** is introducing **support for Change Data Capture (CDC)**: This makes implementing a popular streaming architecture much easier: The stream processor is constantly reading the internal transaction log of a traditional database (Postgres, MySQL) for heavy analytics workloads.
- **Flink SQL** is also introducing support for **Python User-defined-functions**. You can now use Flink SQL to prepare (join, clean, aggregate) your data, then process it with your own Python code -- no more need to write Java or Scala.

This list is just a selection of some highlights of the release. Make sure to check out the full description of all new features below.
```